src/lib/Recorder.ts

```diff
@@ -1,55 +1,64 @@
+/* eslint-disable max-statements */
+/* eslint-disable max-len */
 import PipeSource from './PipeSource';
 
 type TRecorderOptions = {
 	bufferSize?: number;
 	sampleRate?: number;
 	channelCount?: number;
+	publishFrequency?: number;
 };
 
 const defaultOptions: TRecorderOptions = {
 	bufferSize: 1024,
 	sampleRate: 44100,
 	channelCount: 1,
+	publishFrequency: 200,
```

Comment:
I assume these are ms, so can we call this publishInterval to make that more obvious?
"frequency" means "how many times". "interval" means "every how much time".